Kotlin/src/main/kotlin/com/gildedrose/TexttestFixture.kt

```diff
@@ -5,16 +5,16 @@ fun main(args: Array<String>) {
     println("OMGHAI!")
 
     val items = listOf(
-        Item("+5 Dexterity Vest", 10, 20), //
-            Item("Aged Brie", 2, 0), //
-            Item("Elixir of the Mongoose", 5, 7), //
-            Item("Sulfuras, Hand of Ragnaros", 0, 80), //
-            Item("Sulfuras, Hand of Ragnaros", -1, 80),
-            Item("Backstage passes to a TAFKAL80ETC concert", 15, 20),
-            Item("Backstage passes to a TAFKAL80ETC concert", 10, 49),
-            Item("Backstage passes to a TAFKAL80ETC concert", 5, 49),
+        ItemType("+5 Dexterity Vest", 10, 20, false), //
+        ItemType("Aged Brie", 2, 0, false), //
+        ItemType("Elixir of the Mongoose", 5, 7, false), //
+        ItemType("Sulfuras, Hand of Ragnaros", 0, 80, false), //
+        ItemType("Sulfuras, Hand of Ragnaros", -1, 80, false),
+        ItemType("Backstage passes to a TAFKAL80ETC concert", 15, 20, false),
+        ItemType("Backstage passes to a TAFKAL80ETC concert", 10, 49, false),
+        ItemType("Backstage passes to a TAFKAL80ETC concert", 5, 49, false),
             // this conjured item does not work properly yet
-            Item("Conjured Mana Cake", 3, 6)
+        ItemType("Conjured Mana Cake", 3, 6, true)
```

Comment:
I can see why you've done this, and that the spec doesn't explicitly forbid modifying this file, but this is the file it means when referring to the `items property` (and I fully acknowledge that this isn't close to being obvious!)

I'd suggest reverting this file, and thinking about how you'd implement the main logic when Items are passed in using the Item class :)